projects/bp-gallery/src/components/views/picture/sidebar/picture-info/DateRangeSelectionField.tsx

```diff
@@ -57,6 +59,15 @@ const DateRangeSelectionField = ({
     setAnchorElement(anchor);
   };
 
+  const handleChange = () => {
```

Comment:
Vielleicht von der Benennung eher handleCheckBoxChange da es spezifisch ist

Reply:
I think `handleIsEstimateChange` would be even better.